packages/components/src/Chip/components/ChipPrefix/Chip.Prefix.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { PropsWithChildren } from "react";
+import { useChildComponent } from "@jobber/hooks/useChildComponent";
+import classNames from "classnames";
+import styles from "../../Chip.css";
```

Comment:
This was a deliberate choice to cut down on duplication of styles because Prefix/Suffix share a few, so they're doubled up/cascading in the main file. If I moved the CSS for Prefix in, I would have to move the CSS for Suffix as well, and then I'd have to duplicate the styles which I wanted to avoid.

Reply:
that makes sense